spec-draft.md

```diff
@@ -272,10 +272,65 @@ as follows:
 * The virtual keyboard: use sendKeys
 * acceleromator: TODO @mdas is working on this
 * geolocation: use regular webdriver endpoints
-* battery level: not in spec, perhaps exposed via executeScript
 * network speed: not in spec, perhaps exposed via executeScript
 
 
+Battery
+-------
+Device battery changes might trigger events on users applications, controlling this will help monitor and verify how apps responds to these factors.
+
+The battery state should have two options: ON, OFF depending if the power ac state it's charging or not.
+
+* GET /session/:sessionId/appium/device/battery_state
+  * returns BatteryState
+
+
+* POST /session/:sessionId/appium/device/battery_state
+  * accepts BatteryState
+
+
+A `BatteryState` can be one of two strings:
+ * ON
+ * OFF
+
+In some cases, apps might respond different on low battery level so the idea is to provide a method to set specific battery level for the devices.
```

Comment:
maybe mention that this is percent? and should we state that it is a range of integers (rather than real values?).